corehq/apps/domain/templates/domain/manage_releases.html

```diff
@@ -0,0 +1,64 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+{% requirejs_main 'app_manager/js/manage_releases' %}
+
+{% block page_content %}
+    {% initial_page_data 'enabled_app_releases' enabled_app_releases %}
+    {% registerurl 'deactivate_release_restriction' domain '---'%}
+    {% registerurl 'activate_release_restriction' domain '---'%}
+    <div class="row">
+        <div class="col-sm-12">
+            <div class="panel panel-modern-gray panel-form-only">
+                <div class="panel-body">
+                    <form class="form-horizontal disable-on-submit" id="manage-app-releases" action=""
```

Comment:
Huh, I didn't know `disable-on-submit` worked on forms, I thought it had to be applied to the buttons themselves. Cool.